packages/components/src/components/modals/DialogModal/DialogModal.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
```

Comment:
nit: can you please import just ReactNode

Reply:
e1b9c0e5d6